index.js

```diff
@@ -57,6 +73,34 @@ router
     ctx.status = 202
     ctx.body = 'OK'
   })
+  .get('/cloudflareError.png', async ctx => {
+    if (ctx.query['cloudflareErrorType'] == undefined || ctx.query['rayID'] == undefined || ctx.query['clientIP'] == undefined) {
+      ctx.status = 404
+    }
+    else if (!['500', '1000'].includes(ctx.query['cloudflareErrorType'])) {
```

Comment:
If you care about being the fastest, either move this array as a constant into the module scope, or do 2 separate checks.